framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -58,7 +62,10 @@ public static void main(String[] args) {
         System.exit(exitStatus);
     }
 
-    /** The path to the annotated jdk jar to use. */
+    /**
+     * The path to the annotated jdk jar to use in case of Java8 or directory containing
```

Comment:
End sentences with a period.  State more clearly that this is one of two things; for example, change "or" to ", or to a".

Reply:
Done